src/Library/demos/Toolbar View/main.blp

```diff
@@ -0,0 +1,220 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.Window {
+  width-request: 360;
+  height-request: 200;
+  default-width: 640;
+  default-height: 640;
+
+  Adw.ToolbarView toolbar_view {
+    vexpand:true;
+    extend-content-to-bottom-edge: true;
+    extend-content-to-top-edge: true;
+    top-bar-style: bind barstyle_select.selected;
+    bottom-bar-style: bind barstyle_select.selected;
+    reveal-top-bars: false;
+    reveal-bottom-bars: false;
+    [top]
+    Adw.HeaderBar header_top {
+      title-widget: Label{
+      label:_("Header Bar");
+      };
+    }
+    content: Adw.StatusPage {
+      Box {
+        halign: center;
+        orientation: vertical;
+        spacing: 18;
+        Label{
+          label: _("Toolbar View");
+          styles ["title-1"]
+        }
+        Label{
+          wrap: true;
+          label: _("A widget containing a page, as well as top and/or bottom bars.");
+        }
+        LinkButton{
+          label: _("API Reference");
+          uri: "https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.ToolbarView.html";
+        }
+        Box {
+          halign: center;
+          Adw.PreferencesGroup {
+            styles ["boxed-list"]
+            Adw.ComboRow topbar_select {
+              title: _("Top Bar :");
+              model: StringList {
+                strings ["Header Bar","Tab Bar","View Switcher Bar","Action Bar","PopOverMenu Bar","Search Bar","Gtk Box"]
```

Comment:
I'm not sure if we're going to keep these, but they should be marked for `_("Translation")`, if so.

Reply:
We are keeping them, it was part of the specs.

https://github.com/sonnyp/Workbench/issues/408#issuecomment-1637150356